src/util/unicode.cpp

```diff
@@ -253,3 +255,37 @@ const char **narrow_argv(int argc, const wchar_t **argv_wide)
 
   return argv_narrow;
 }
+
+std::wstring utf8_to_utf16_big_endian(const std::string& in)
+{
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t> > converter;
+  return converter.from_bytes(in);
+}
+
+std::wstring utf8_to_utf16_little_endian(const std::string& in)
+{
+  const std::codecvt_mode mode=std::codecvt_mode::little_endian;
+  const unsigned long maxcode=0x10ffff;
```

Comment:
I added a comment and a reference to the c++ documentation: https://github.com/diffblue/cbmc/pull/372/commits/ccbe6f77930851e2ca27986c0b91f1210c28ec3e